docs/pages/machine-id/guides/jenkins.mdx

```diff
@@ -0,0 +1,149 @@
+
+## Prerequisites
+
+You will need the following tools to use Teleport with Jenkins.
+
+- [Teleport Enterprise](../../enterprise/introduction.mdx) or [Teleport Cloud](../../cloud/introduction.mdx) >= 9.0.0 
+- `ssh` OpenSSH tool
+- `Jenkins`
+
+In addition, if you already have not done so, follow the
+[Machine ID Getting Started Guide](../../docs/pages/machine-id/getting-started.mdx)
+to create a bot user and start Machine ID.
+
+## Architecture
+
+Before we begin, it should be noted that Jenkins is a tool that is notoriously
+difficult to secure. Machine ID is one part of securing your infrastructure,
+but it alone is not sufficient. Below we will provide some basic guidance which
+can help improve the security posture of your Jenkins install.
+
+The simplest Jenkins deployments have the controller and agents run on the
+same host. This deployment model is simple to get started with, however any
+compromise of the `jenkins` user within a single pipeline can lead to the
+compromise of your entire CI/CD infrastructure.
+
+A slightly more complex, but more secure, deployment is running your Jenkins
+controllers and agents on different hosts and pinning workloads to specific
+agents. This is a improvement over the simple deployment because you can limit
+the blast radius of the compromise of a single pipeline to a subset of your
+CI/CD infrastructure instead of all of your infrastructure.
+
+We strongly encourage the use of the second deployment model whenever possible,
+with ephemeral hosts when possible. When using Machine ID with this model,
+create and run Machine ID bots per-pipeline. This will allow you to give each
+pipeline the minimal set of server access, reduce the blast radius if one
+pipeline is compromised, and allow you to remotely audit and lock pipelines if
+you detect malicious behavior.
+
+<Figure align="left" bordered caption="Jenkins Deployments">
+  ![Jenkins Deployments](../../img/machine-id/machine-id-jenkins.png)
+</Figure>
+
+## Step 1/2 Configure and start Machine ID
+
+Machine ID allows you to use Linux ACLs to control access to certificates on
+disk. You will use this to limit the access Jenkins has to the short-lived
+certificates Machine ID issues.
+
+In the example that follows, we'll work with the assumption you will be running
+Machine ID as the Linux user `teleport` and Jenkins as the Linux user
+`jenkins`. Create and initialize the directories you will need using the
+`tbot init` command.
+
+```code
+$ tbot init \
+    --destination-dir=/opt/machine-id \
+    --bot-user=teleport \
+    --owner=teleport:teleport \
+    --reader-user=jenkins \
+```
+
+Next, you will use systemd to run Machine ID in the background of each
+Jenkins worker.
+
+First create a configuration file for Machine ID at `/etc/tbot.yaml`.
+
+<Details scope={["cloud"]} scopeOnly={true}>
+```yaml
+auth_server: "example.teleport.sh:443"
+onboarding:
+  join_method: "token"
+  token: "00000000000000000000000000000000"
```

Comment:
It might be good to clarify somewhere that we'll have different tokens (and for that matter, a completely separate bots) for each worker. I know it's alluded to in the previous step, but seemed easy to glance over IMO.